var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -93,6 +93,23 @@ def configure_args(self):
             '-Dloclibpth=' + self.spec['gdbm'].prefix.lib,
         ]
 
+        # Extensions are installed into their private tree via
+        # `INSTALL_BASE`/`--install_base` (see [1]) which results in a
+        # "predictable" installation tree that sadly does not match the
+        # Perl core's @INC structure.  This means that when activation
```

Comment:
As a side effect of being able to support many old releases and many OS's and, well, just being Perl I guess..., the things in the core are complicated.

The structure of additional packages, even added by the standard tool chains, is also complicated.

I think that this is a reasonably simple solution that deals with the problem without adding too much complexity.

We could install into either the `site` or `vendor` subtrees, (search [here](http://search.cpan.org/~bingos/ExtUtils-MakeMaker-7.30/lib/ExtUtils/MakeMaker.pm) for "make install" for a bit of background) but other people sometimes have designs on them.  Carving out our own little slice of heaven seems reasonable.